crates/viewer/re_dataframe_ui/src/table_blueprint.rs

```diff
@@ -158,32 +157,54 @@ pub fn default_display_name_for_column(desc: &ColumnDescriptorRef<'_>) -> String
         ColumnDescriptorRef::RowId(_) | ColumnDescriptorRef::Time(_) => desc.display_name(),
```

Comment:
I'm not sure why this is using `display_name` and the code below uses `column_name`… but that's for another day